src/containers/App/index.js

```diff
@@ -20,6 +20,11 @@ const RESOLVED_CALLBACK_DELAY = process.env.NODE_ENV === 'development' ? LOADER_
 const App = () => {
   const [isLoading, setIsLoading] = React.useState(true);
 
+  React.useEffect(() => {
```

Comment:
What is this for?

Reply:
Some components make the app X scrollable. this locks the screen making sure it is not allowed

Reply:
@jorgebaralt Which components are those? We should fix that instead, this seems like a bandaid fix.